apps/ledger-live-mobile/src/screens/AddAccounts/02-SelectDevice.tsx

```diff
@@ -107,20 +90,13 @@ export default function AddAccountsSelectDevice({ navigation, route }: Props) {
       ]}
     >
       <SkipSelectDevice route={route} onResult={setDevice} />
-      {newDeviceSelectionFeatureFlag?.enabled ? (
-        <Flex px={16} py={8} flex={1}>
-          <SelectDevice2
-            onSelect={setDevice}
-            stopBleScanning={!!device || !isFocused}
-            requestToSetHeaderOptions={requestToSetHeaderOptions}
-          />
-        </Flex>
-      ) : (
-        <NavigationScrollView style={styles.scroll} contentContainerStyle={styles.scrollContainer}>
-          <TrackScreen category="AddAccounts" name="SelectDevice" currencyName={currency?.name} />
-          <SelectDevice onSelect={onSetDevice} />
-        </NavigationScrollView>
-      )}
+      <Flex px={16} py={8} flex={1}>
+        <SelectDevice2
```

Comment:
[ASK] Is `SelectDevice` component used anymore ? Can we remove it ? Can we rename `SelectDevice2` to `SelectDevice` ?

Reply:
I think it's still in use for a debug need, I'll check it and treat it in a more dedicated PR if needed.